src/v2/models/AppState.js

```diff
@@ -54,10 +54,10 @@ export default Model.extend({
     showSignoutLink: {
       deps: ['idx'],
       fn: function (idx = {}) {
-        const invalidSignOutSteps = ['IDENTIFY', 'ENROLL', 'SUCCESS'];
-        // hide signout for IDENTIFY, ENROLL & SUCCESS step
+        const invalidSignOutSteps = ['IDENTIFY', 'LOGIN', 'ENROLL', 'SUCCESS'];
```

Comment:
Note: The `intent` here (seen through debugger) seems to be `LOGIN` and not `ENROLL`. However, for `LOGIN` we don't need to show sign out link as well?